request.cgi

```diff
@@ -248,7 +249,7 @@ sub queue {
                 requesters.login_name, requestees.realname,
                 requestees.login_name, flags.modification_date, attachments.ispatch
                 cclist_accessible, bugs.reporter, bugs.reporter_accessible,
-                bugs.assigned_to, attachments.ispatch');
+                bugs.assigned_to, attachments.mimetype');
```

Comment:
doesn't this need to also group by ispatch?

Reply:
There's already `ispatch` 2 lines above. Looks like it's a duplicate, so just changing it to `mimetype`.